tests/rados/test_bug_fixes.py

```diff
@@ -298,3 +302,137 @@ def run(ceph_cluster, **kw):
+            osd_list = rados_obj.run_ceph_command(cmd="ceph osd ls")
+            log.debug(f"List of active OSDs: \n{osd_list}")
+
+            # check the current value of 'bluestore_min_alloc_size'
+            min_alloc_size = int(
+                mon_obj.show_config(
+                    daemon="osd", id=osd_list[0], param="bluestore_min_alloc_size_hdd"
+                )
+            )
+            log.info(
+                f"Current persistent value of 'bluestore_min_alloc_size_hdd' is {min_alloc_size}"
+            )
+            min_alloc_size_ssd = int(
+                mon_obj.show_config(
+                    daemon="osd", id=osd_list[0], param="bluestore_min_alloc_size_ssd"
+                )
+            )
+            log.info(
+                f"Current persistent value of 'bluestore_min_alloc_size_ssd' is {min_alloc_size}"
+            )
+
+            # cherry-pick one OSD ID from each osd node
+            test_osd_list = [
+                rados_obj.collect_osd_daemon_ids(osd_node=node)[0] for node in osd_nodes
+            ]
+
+            log.info(
+                f"OSDs for which bluefs_shared_alloc_size will be modified: {test_osd_list}"
+            )
+            shared_alloc_size = int(min_alloc_size / 2)
+            log.info(
+                f"bluefs_shared_alloc_size parameter will be set to - {shared_alloc_size}"
+            )
+
+            for osd_id in test_osd_list:
+                mon_obj.set_config(
+                    section=f"osd.{osd_id}",
+                    name="bluefs_shared_alloc_size",
+                    value=shared_alloc_size,
+                )
+
+            time.sleep(5)
+
+            # restart OSD orch service
+            assert rados_obj.restart_daemon_services(daemon="osd")
+
+            # determine how much % cluster is already filled
+            current_fill_ratio = rados_obj.get_cephdf_stats()["stats"][
+                "total_used_raw_ratio"
+            ]
+
+            # determine the number of objects to be written to the pool
+            # to achieve 70% utilization
+            osd_df_stats = rados_obj.get_osd_df_stats(
+                tree=False, filter_by="name", filter="osd.0"
+            )
+            osd_size = osd_df_stats["nodes"][0]["kb"]
+            num_objs = int(
+                (osd_size * (0.7 - current_fill_ratio) / bench_obj_size_kb) + 1
+            )
+
+            # create a sample pool to write data
+            assert rados_obj.create_pool(pool_name=_pool_name)
+
+            # fill the cluster till 70% capacity
+            bench_config = {
+                "seconds": 300,
+                "b": f"{bench_obj_size_kb}KB",
+                "no-cleanup": True,
+                "max-objects": num_objs,
+            }
+            bench_obj.write(client=client_node, pool_name=_pool_name, **bench_config)
+
+            time.sleep(10)
+
+            # restart OSD orch service
+            assert rados_obj.restart_daemon_services(daemon="osd")
+
```

Comment:
where are we checking for crashes post reboot of OSDs?

Reply:
Thank you! I have added code to check for crashes